doc/nrf/ug_nrf9160.rst

```diff
@@ -174,15 +174,14 @@ Complete the following steps to provision the certificate:
 Build targets
 *************
 
+Make sure to select a suitable build target when building your application.
+
 In Zephyr, :ref:`zephyr:nrf9160dk_nrf9160` is divided into two different build targets:
```

Comment:
I think we should maybe either mention firmware in this introductory sentence, or keep firmware in the bullet points :)